packages/sdk/src/entities/BorrowingVault.ts

```diff
@@ -157,10 +180,12 @@ export class BorrowingVault {
   /**
    * Creates a connection by setting an rpc provider.
    *
-   * @param configParams - {@link ConfigParams} object with infura and alchemy ids
+   * @param configParams - {@link ChainConfigParams} object with infura and alchemy ids
    */
-  setConnection(configParams: ConfigParams): BorrowingVault {
-    this.rpcProvider = Config.rpcProviderFrom(configParams, this.chainId);
+  setConnection(configParams: ChainConfigParams): BorrowingVault {
```

Comment:
Maybe `ChainConfigParam` could be shortened with `ChainConfig` (almost the same meaning 🤔 )

Reply:
done